desktop/apps/editorial_features/components/venice_2017/templates/footer.jade

```diff
@@ -1,2 +1,16 @@
+if sub_articles.length > 3
+  - length = 'over'
+else
+  -length = sub_articles.length
+
 .venice-footer__articles
-	h2.title The Venice Biennale
\ No newline at end of file
+  h2.title The Venice Biennale
+  .venice-footer__article-carousel(data-length=length)
+    .mgr-horizontal-nav
+      .mgr-cells.js-mgr-cells
+        each article in sub_articles
+          .mgr-cell.js-mgr-cell
+            a(href="#{sd.APP_URL}/article/#{article.slug}")
+              img(src=crop(article.thumbnail_image, {width: 400, height: 265}))
```

Comment:
333 x 213 -- i think i just used a calculator to scale up, but can adjust to 3x2 which is closest